lib/include/openamp/rpmsg.h

```diff
@@ -81,12 +81,19 @@ struct rpmsg_endpoint {
 
 /**
  * struct rpmsg_device_ops - RPMsg device operations
- * @send_offchannel_raw: send RPMsg data
+ * @hold_rx_buffer: hold RPMsg RX buffer
+ * @release_rx_buffer: release RPMsg RX buffer
+ * @get_tx_payload_buffer: get RPMsg TX buffer
+ * @send_offchannel_nocopy: send RPMsg data without copy
  */
 struct rpmsg_device_ops {
-	int (*send_offchannel_raw)(struct rpmsg_device *rdev,
-				   uint32_t src, uint32_t dst,
-				   const void *data, int len, int wait);
+	void (*hold_rx_buffer)(struct rpmsg_device *rdev, void *rxbuf);
```

Comment:
keep send_offchannel_raw ops. we have to preserve this API for sustainability. then to limit lib footprint impact. it will more simple to split legacy and no copy support.